src/sockets.c

```diff
@@ -1,38 +1,18 @@
-/* A minimal wrapper for socket communication.
-
-Copyright (C) 2013, Joshua More and Michele Ceriotti
```

Comment:
Do we have an authoritative AUTHORS list? We seem to be referring to cp2k.org, but I couldn't find it on a quick glance. Otherwise without an authoritative AUTHORS file where both of them are explicitly listed, we're essentially removing them for the list of copyright holders.

Reply:
No we don't maintain an AUTHORS list. The authorship is partially documented in the Doxygen boxes and perfectly tracked by the git history.

> we're essentially removing them for the list of copyright holders.

This I don't understand. The copyright assignment doesn't change just because we document it differently.

Reply:
https://reuse.software/faq/#copyright-holder-author

Reply:
> This I don't understand. The copyright assignment doesn't change just because we document it differently.

based on the above we are unfortunately not allowed to document it differently